resources/windows/wix/customactions.wxs

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="Include_CustomActions" />
+
+    <CustomAction Id="SaveCmdLineInstallDir"
+      Property="CMDLINE_INSTALLDIR"
+      Value="[INSTALLDIR]"
+      Execute="firstSequence" />
+    <CustomAction Id="SetFromCmdLineInstallDir"
+      Property="INSTALLDIR"
+      Value="[CMDLINE_INSTALLDIR]"
+      Execute="firstSequence" />
```

Comment:
No reason why we can't take these lines + any others that are commonly used and put them in a customactions-common.wxs file in vanagon.
As noted earlier, its really the sequence files are better entirely specific to the project.